pkg/core/config/main.go

```diff
@@ -47,13 +51,15 @@ var (
 
 // Config contains cli configuration
 type Config struct {
-	Name       string
-	PipelineID string        // PipelineID allows to identify a full pipeline run, this value is propagated into each target if not defined at that level
-	Title      string        // Title is used for the full pipeline
-	Source     source.Config // **Deprecated** 2021/02/18 Is replaced by Sources, this setting will be deleted in a future release
-	Sources    map[string]source.Config
-	Conditions map[string]condition.Config
-	Targets    map[string]target.Config
+	Name         string
+	PipelineID   string                        // PipelineID allows to identify a full pipeline run, this value is propagated into each target if not defined at that level
+	Title        string                        // Title is used for the full pipeline
+	Source       source.Config                 // **Deprecated** 2021/02/18 Is replaced by Sources, this setting will be deleted in a future release
```

Comment:
Removal planned for the next big release?

Reply:
That's a good suggestion, I don't have a strong opinion about when to fully remove it. It will be almost one year since I deprecated that syntax.
We also have the postfix/prefix and now the scm to deprecated

Reply:
At this stage we have the two following issues, I need to create a new issue for the deprecation introduced in this PR, using the label [cleanup](https://github.com/updatecli/updatecli/labels/cleanup)